src/shared/hooks/use-draggable-list-modal.tsx

```diff
@@ -34,7 +35,7 @@ export function useDraggableListModal(config?: {
 		<DraggableListModal
 			{...config?.modal}
 			renderItem={(item) => <DraggableBlock block={item} />}
-			isOpen={isOpen}
+			isOpen={!isNil(config?.modal?.isOpen) ? config?.modal?.isOpen || false : isOpen} // Allow external config to open modal, if not provided, internal isOpen state will be used
```

Comment:
You can achieve this by moving `isOpen` above the spread-operator.

Reply:
clean